code/ship/shipfx.cpp

```diff
@@ -1230,33 +1230,16 @@ void shipfx_emit_spark( int n, int sn )
 		vm_vec_add2(&outpnt,&obj->pos);
 	}
 
-	//WMC - I have two options here:
```

Comment:
Removing these comments is the right move, IMO, but others might see it differently.

Reply:
phreak implemented the very thing WMC was stressing over later, so it really doesn't make much sense to leave the comment here as it's referring to an older version of the warpeffect system.